src/main/resources/hudson/tasks/junit/JUnitResultArchiver/config.jelly

```diff
@@ -30,8 +30,12 @@ THE SOFTWARE.
         description="${%description('http://ant.apache.org/manual/Types/fileset.html')}">
         <f:textbox />
     </f:entry>
-    <f:entry field="keepLongStdio" title="">
-        <f:checkbox name="keepLongStdio" title="${%Retain long standard output/error}"/>
+    <f:entry field="stdioRetention" title="">
+        <select name="stdioRetention">
+            <option value="none">None</option>
+            <option value="failed">Failed tests only</option>
+            <option value="all">All</option>
+        </select>
```

Comment:
these should be fetched from code with a `doFillXXX` method using an `f:select`.

Reply:
I've taken a stab at this; let me know how it looks. I'm also not terribly clear on how to test this (I've been using a `Jenkinsfile` to configure the plugin for testing); if there's a resource out there on how to validate this, please let me know.

Reply:
Never mind, figured out how to test (configure -> Post-build Actions -> Publish JUnit test result report).